app/src/main/java/com/bogdan/codeforceswatcher/network/RestClient.kt

```diff
@@ -21,7 +27,7 @@ object RestClient {
 
         return@lazy Retrofit.Builder()
             .baseUrl("http://www.codeforces.com/api/")
-            .client(okHttpClient)
+            .client(okHttpClient.build())
```

Comment:
If `build()` is called only here, then `okHttpClient` isn't really OkHttpClient anymore, but rather its build. So it's better to call build when initializing `OkHttpClient` in the first place.